### PR TITLE
chore: governance url, git dependency compatibility

### PR DESCRIPTION
**Pull request by thepiwo**

### CODE REVIEW

File: src/views/Governance.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <iframe
     class="governance"
-    src="https://governance.aeternity.art/#/?iframe=1"
```
[the remark this answers is not in the document]
Comment (thepiwo) [reply src/views/Governance.vue:4]: doesn't work with it, no clue

[the remark this answers is not in the document]
Comment (thepiwo) [reply src/views/Governance.vue:4]: seems to work now, no clue why it didn't earlier

[the remark this answers is not in the document]
Comment (kenodressel) [reply src/views/Governance.vue:4]: resolve, thx :)

**Review by kenodressel — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     "@popperjs/core": "^2.9.2",
     "@uvue/core": "^0.1.7",
     "@uvue/server": "^0.1.7",
-    "aeternity-fungible-token": "github:aeternity/aeternity-fungible-token#v2.0.0",
+    "aeternity-fungible-token": "git+https://git@github.com/aeternity/aeternity-fungible-token#v2.0.0",
```
Comment (davidyuk) [inline package.json:33]: is `github:` prefix not working in some case?

Comment (thepiwo) [reply package.json:33]: I had some issues with it in installs for some reason, no clue why

**Pull request merged**